mmv1/products/compute/RegionCommitment.yaml

```diff
@@ -156,17 +156,15 @@ properties:
           name: 'acceleratorType'
           description: |
             Name of the accelerator type resource. Applicable only when the type is ACCELERATOR.
-  - !ruby/object:Api::Type::Enum
+  - !ruby/object:Api::Type::String
     name: 'type'
-    default_from_api: true
+    output: true
```

Comment:
I think this should still default to the API value. Changing it to `output` here will make it unsettable by the user.

Reply:
Got  it, I am reverting this change...

Reply:
thanks!